src/SqmHelpers/SqmCommandFacadeGenerator.ttinclude

```diff
@@ -17,9 +17,9 @@
 //                                                                           
 // This file contains SQM trace logging methods.
 
-using EnvDTE;
```

Comment:
Stylistic tweaks to the code generation template to make the generated output match the expected style (the generated code had been manually edited, so those manual changes were lost when I regenerated the code).